testing/wireguard-tools/APKBUILD

```diff
@@ -10,6 +10,7 @@ arch='all'
 url='https://www.wireguard.com'
 license="GPL-2.0"
 makedepends="libmnl-dev"
+depends="bash iproute2 procps"
```

Comment:
does it really require `bash`?

Reply:
`wg-quick` specifies:

```
#!/bin/bash
# SPDX-License-Identifier: GPL-2.0
```

& makes use of `bash` arrays in various places:
```
LXC-alpine [~]$ grep "]}" $(which wg-quick)
SELF="$(readlink -f "${BASH_SOURCE[0]}")"
	INTERFACE="${BASH_REMATCH[2]}"
	[[ $UID == 0 ]] || exec sudo -p "$PROGRAM must be run as root. Please enter the password for %u to continue: " -- "$BASH" -- "$SELF" "${ARGS[@]}"
		output="$(ip route get "${BASH_REMATCH[1]}" || true)"
		[[ ( $output =~ mtu\ ([0-9]+) || ( $output =~ dev\ ([^ ]+) && $(ip link show dev "${BASH_REMATCH[1]}") =~ mtu\ ([0-9]+) ) ) && ${BASH_REMATCH[1]} -gt $mtu ]] && mtu="${BASH_REMATCH[1]}"
		[[ ( $output =~ mtu\ ([0-9]+) || ( $output =~ dev\ ([^ ]+) && $(ip link show dev "${BASH_REMATCH[1]}") =~ mtu\ ([0-9]+) ) ) && ${BASH_REMATCH[1]} -gt $mtu ]] && mtu="${BASH_REMATCH[1]}"
		echo "${BASH_REMATCH[1]}." && return 0
	[[ ${#DNS[@]} -gt 0 ]] || return 0
	printf 'nameserver %s\n' "${DNS[@]}" | cmd resolvconf -a "$(resolvconf_iface_prefix)$INTERFACE" -m 0 -x
	[[ ${#DNS[@]} -gt 0 ]] || return 0
	for address in ${BASH_REMATCH[1]}; do
		[[ $address =~ ^nameserver\ ([a-zA-Z0-9_=+:%.-]+)$ ]] && new_config+="DNS = ${BASH_REMATCH[1]}"$'\n'
	[[ -n $MTU && $(ip link show dev "$INTERFACE") =~ mtu\ ([0-9]+) ]] && new_config+="MTU = ${BASH_REMATCH[1]}"$'\n'
	for cmd in "${PRE_UP[@]}"; do
	for cmd in "${POST_UP[@]}"; do
	for cmd in "${PRE_DOWN[@]}"; do
	for cmd in "${POST_DOWN[@]}"; do
	execute_hooks "${PRE_UP[@]}"
	for i in "${ADDRESSES[@]}"; do
	execute_hooks "${POST_UP[@]}"
	execute_hooks "${PRE_DOWN[@]}"
	execute_hooks "${POST_DOWN[@]}"
```